src/main/java/ulcrs/Server.java

```diff
@@ -0,0 +1,66 @@
+package ulcrs;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Spark;
+import ulcrs.controllers.CourseController;
+import ulcrs.controllers.ScheduleController;
+import ulcrs.controllers.SessionController;
+import ulcrs.controllers.ShiftController;
+import ulcrs.controllers.TutorController;
+
+public class Server {
+
+    private static Logger log = LoggerFactory.getLogger(Server.class);
+    private TutorController tutorController;
+    private CourseController courseController;
+    private ScheduleController scheduleController;
+    private ShiftController shiftController;
+    private SessionController sessionController;
+
+    public static void main(String args[]) {
+        Server server = startServer();
+
+        // Route to the proper methods
+        Spark.path("/ulcrs", () -> {
+            // Paths for resources, handled by appropriate Controllers
+            Spark.path("/tutor", server.tutorController.routes());
+            Spark.path("/course", server.courseController.routes());
+            Spark.path("/schedule", server.scheduleController.routes());
+            Spark.path("/shift", server.shiftController.routes());
+            Spark.path("/session", server.sessionController.routes());

```

Comment:
For these Spark methods, Spark recommends you use static methods. This isn't vital as it still works, but it's something to consider